Sources/Exporters/OpenTelemetryProtocol/metric/OtlpMetricExporter.swift

```diff
@@ -16,17 +17,19 @@ public class OtlpMetricExporter: MetricExporter {
     let config : OtlpConfiguration
     var callOptions : CallOptions? = nil
 
+    
 
-
-    public init(channel: GRPCChannel, config: OtlpConfiguration = OtlpConfiguration()) {
+    public init(channel: GRPCChannel, config: OtlpConfiguration = OtlpConfiguration(), logger: Logger = Logger(label: "io.grpc", factory: { _ in SwiftLogNoOpLogHandler() }), envVarHeaders: [(String,String)]? = EnvVarHeaders.attributes) {
```

Comment:
Signature changed to include the logger and envVarHeaders.  Both include default values that match the older behavior. This is a non-breaking change for existing clients.